### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by medvednikov** [eul-im/eul#131]
Volt uses native UI, but accessibility is not great right now. Chat text is not readable, neither is the "add an account" window.

It has to be 100% accessible.

**Comment by mikolysz** [comment]
How far are you willing to go with this? Making the controls accessible is one thing and it's certainly the top priority, but stuff like automatically reading incoming messages in the currently focused chat window would also be useful. Perhaps some keyboard shortcuts like ctrl+[1-0] that would "send" one of the recent messages to the screenreader for quick reading?

**Comment by medvednikov** [comment]
Sounds good, I'll consult with you once I start working on this.

**Comment by Menelion** [comment]
@medvednikov Found your client and would like to try it, but there is no Windows build at the moment on your site, it seems. Anyways, if you need any accessibility testing and/or help, please don't hesitate to contact me or simply comment in this issue. Thanks so much for your great effort, I'll try the client as soon as it is available again. Thanks!
